torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -277,6 +278,7 @@ class ScriptModuleSerializer {
 
   void writeByteCode(const Module& module) {
     std::vector<c10::IValue> elements;
+    elements.emplace_back(Tup({BYTECODE_VERSION_STR, BYTECODE_VERSION_NUMBER}));
```

Comment:
This format it weird to me because it takes version with a key but doesn't take the list of elements with a key (and then inlines the list of elements into a tuple with the version tuple. Is this suppose to be a named table of things? or a list of things? This mixes both.
`(("version", version), <elements>)`?